external/CMakeLists.txt

```diff
@@ -35,6 +35,21 @@ if(NOT N EQUAL 0)
     endif()
 endif()
 
+# Setup versions for external project 
+set(EDGE_SDK_CPP_GOOGLETEST_URL "https://github.com/google/googletest.git")
+set(EDGE_SDK_CPP_GOOGLETEST_TAG "6d668180665e313bbda19f8187abf6140e80a7d7")
```

Comment:
I find it strange to have an SHA hash instead of a tag.
Please change to "release-1.10.0" or to " release-1.8.1" if it does not build with 1.10.0.